client/src/components/EmailFrontend/EmailFrontend.js

```diff
@@ -162,20 +164,22 @@ class EmailFrontend extends React.Component {
 
     const { include_privacy } = this.props;
 
-    const user_fields = _.omitBy(
-      template,
-      ({ form_type }, key) => key === "meta" || !form_type
-    );
+    const user_fields = _.omitBy(template, ({ form_type }, key) => {
+      key === "meta" || !form_type;
+    });
```

Comment:
Some how this got mangled. Look closely, the applied function went from a one-liner to having a body (but no return function). Nothing was being omitted as a result, and the future logic relying on `user_fields` was broken. That's why the submit button wasn't working as expected.

Reply:
Ohh I see now, thanks for pointing that out!